lib/bloc/network/network_settings_bloc.dart

```diff
@@ -84,9 +84,13 @@ class NetworkSettingsBloc extends Cubit<NetworkSettingsState> with HydratedMixin
   }
 
   Future<bool> _testUri(Uri uri) async {
+    // We need to make sure that the mempool rest api is supported
+    // as the sdk depends on it.
+    uri = Uri.parse("${uri.toString()}/api/v1/fees/recommended");
+
     try {
```

Comment:
I think it is better to break it into two methods; ` _testUri(Uri uri)` continues as it is without changing the URI and you create a second `_testUriSupportsExplorerApi(Uri uri)` (or something like that) where you call testUri from /recommended and eventually other endpoints

Reply:
Done